app/controlplane/api/controlplane/v1/response_messages.proto

```diff
@@ -135,6 +135,7 @@ message PolicyEvaluation {
   PolicyReference policy_reference = 10;
   bool skipped = 12;
   repeated string skip_reasons = 13;
+  repeated string requirements = 14;
```

Comment:
They are not type? just regular strings?

Reply:
yes, they are strings, just like they come from the attestation.